02-express-tutorial/public/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1> Week 3 Express </h1>
+    <button id='getAllProductsBtn'>See All Products</button>
+    <div id="productsContainer"></div>
+
+    <script>
+        document.addEventListener('DOMContentLoaded', async () => {
+            const getAllProductsBtn = document.getElementById('getAllProductsBtn');
+            const productsContainer = document.getElementById('productsContainer');
+
+            getAllProductsBtn.addEventListener('click', async () => {
```

Comment:
Nitpick (doesn't require change):
When using document selectors, it's always best to check if the selector node is defined before using methods. For example, if you happen to misspell an id, then you'll get an error logged to the console that states `can't read property "addEventListener" of undefined`.